src/main/java/com/labC/libraryCheckoutSystem/UserServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.labC.libraryCheckoutSystem;
+
+import org.apache.catalina.User;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Optional;
+
+public class UserServiceImpl implements UserService{
+    @Autowired
+    UserRepo userRepo;
```

Comment:
I'm going to request that you remove the @Autowired and instead utilize contructor dependency injection like so: 

```java

public UserServiceImpl(UserRepo userRepo){
    this.userRepo = userRepo; 
}

```